src/app/examples/chip-example/chip-example.component.html

```diff
@@ -1,2 +1,2 @@
-<kirby-chip [title]="'kirby'"></kirby-chip>
-<kirby-chip [title]="'kirby'" [isActive]="true"></kirby-chip>
+<kirby-chip label="kirby"></kirby-chip>
```

Comment:
Rename to `text` as we use for buttons, badge etc.
```suggestion
<kirby-chip text="kirby"></kirby-chip>
```